### ISSUE TITLE
Delete entire flight plan

### ISSUE THREAD

**Issue by michaelorella** [mockingbirdnest/Principia#2184]
From what I can tell currently, there doesn't seem to be a good way to delete the entire flight plan after you have completed it. Is there functionality that could be added for "time-outs" of maneuvers or a single way to delete the flight plan in entirety?

**Comment by fictionles** [comment]
I've experienced in a few playthroughs that by not deleting a flight plan in a given amount of time will sometimes leave it ghosted and I'll be unable to create a new (or delete an existing one) for the duration of the craft.  The flight plan window would reduce itself to a few pixels in length and width.

**Comment by pleroy** [comment]
@fictionles: The behaviour you are describing (flight plan window shrinking to size 0) is a bug.  We'll need steps to reproduce it (i.e., the sequence of actions that lead to the window shrinking).

**Comment by eggrobin** [comment]
@pleroy 
> The behaviour you are describing (flight plan window shrinking to size 0) is a bug.

It is even a known bug, namely #2158. It will be resolved in Fatou.

**Comment by pleroy** [comment]
@michaelorella: I could see adding a functionality to delete the entire flight plan.  It's not in the current UI to make sure that users don't click it by mistake and lose all their careful tuning, but if/when we revamp the UI it could probably be added (possibly with a confirmation or a 2-click action).

Time-out of manœuvres would cause serious problems from the user experience standpoint.  First, once the first manœuvre is gone the rest of the flight plan would be up in the air with no obvious connection to the current trajectory.  Second, if you screw up a bit in executing the first manœuvre (start a bit after the planned time) then it would vanish and you would have no guidance to help you control your burn.

**Comment by michaelorella** [comment]
@pleroy: I agree it would definitely be painful to lose a carefully planned trajectory and as such would be nice to have some kind of confirmation for a "delete all" type action. My thinking was that in the event that you have executed your flight plan and want to start planning return trips (or something of the sort), it would be nice to have a better way than deleting all of the old maneuvers that you needed to get there. It would definitely be appreciated if this were added to the revamped UI!
